src/index.ts

```diff
@@ -161,9 +177,16 @@ const commands: {[key: string]: (t: browser.tabs.Tab) => Promise<void>} = {
 
 browser.menus.onClicked.addListener((info, tab) => {
     // #cast We only ever create menu items with string IDs
-    const cmd = (<string>info.menuItemId).replace(/^[^:]*:/, '');
-    console.assert(commands[cmd]);
-    commands[cmd](tab).catch(console.log);
+    let cmd = (<string>info.menuItemId).replace(/^[^:]*:/, '');
+    if (cmd.includes(':')) {
+      let folderId;
+      [cmd, folderId] = cmd.split(':');
```

Comment:
I believe bookmark IDs can be arbitrary strings, which means the IDs themselves can contain `:`s. You may truncate part of the folderId if you split this way and the folderId has a `:`.